internal/proxy/proxy_linux.go

```diff
@@ -11,6 +11,10 @@ var exclusionListURLs = []string{
 	"https://raw.githubusercontent.com/anfragment/zen/main/proxy/exclusions/common.txt",
 }
 
+var (
+	ErrUnsupportedDesktopEnvironment = errors.New("system proxy configuration is currently only supported on GNOME")
```

Comment:
This definition needs to be moved to `proxy.go`

Reply:
Got it, fixed.